src/platforms/ios/ioscontroller.swift

```diff
@@ -254,11 +256,55 @@ public class IOSControllerImpl : NSObject {
             }
         }
     }
+    
+    /**
+    * @brief Enables / Disables the Always-On Rule on the runnel
+    * This is used for the "start-on-boot" feature. 
+    * We are setting a rule matching * any network change
+    * which will cause the phone to start the VPN after the phone boots.
+    *
+    * Note: For the Setting to be applied to the iOS settings: 
+    *       tunnel!.saveToPreferences needs to be called.  
```

Comment:
If we need to saveToPreferences afterward, we could include the save as part of this function - to ensure we never forget in the future - and provide this function a completion block to run after the successful save.

Let me know if you want me to write some code for this bit.

Reply:
Good call, added a call to savePrefs but I was not sure how to add this type of handling or generally... really what the controller should do in case this fails :/